go.mod

```diff
@@ -572,6 +572,7 @@ replace (
 	mvdan.cc/lint => mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b
 	mvdan.cc/unparam => mvdan.cc/unparam v0.0.0-20190209190245-fbb59629db34
 	rsc.io/pdf => rsc.io/pdf v0.1.1
+	sigs.k8s.io/apiserver-network-proxy => sigs.k8s.io/apiserver-network-proxy v0.0.0-20200108231339-a03ea77aa169
```

Comment:
do we need to depend on the entire proxy component to pick up the client and proto definition?

Reply:
we don't need to entire proxy component, just a proxy client component and proto component.

I noticed in the vendor directory, it only includes those two components and not the entire package. Is there another approach that you'd suggest?

```
> tree ./vendor/sigs.k8s.io/apiserver-network-proxy
├── LICENSE
├── pkg
│   └── agent
│       └── client
│           ├── BUILD
│           ├── client.go
│           └── conn.go
└── proto
    └── agent
        ├── agent.pb.go
        ├── agent.proto
        └── BUILD
```

Reply:
the only real way to avoid all downstream consumers picking up all the transitive dependencies of the proxy server is to put the client and proto API in a separate module

Reply:
@liggitt: Created a separate module for the network proxy client. We called it `sigs.k8s.io/apiserver-network-proxy/konnectivity-client`. Thank you for your help with the instructions.